packages/components/src/Enumeration/Header/HeaderSelected.component.js

```diff
@@ -0,0 +1,48 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import Action from '../../Actions/Action';
+import theme from './Header.scss';
+
+const headerClasses = () => classNames({
+	[theme['tc-enumeration-header']]: true,
+	'tc-enumeration-header': true,
+});
+
+function HeaderSelected({ headerSelected, selectedItems }) {
+	const getAction = (action, index) => {
```

Comment:
I finally transform into a function declaration to be coherent with other components (headerinput for example), and it is faster too.